fasm2bels/models/gtp_channel_models.py

```diff
@@ -0,0 +1,158 @@
+import re
+import json
+import os
+from .verilog_modeling import Bel, Site, make_inverter_path
+
+ZERO_VAL_PARAMS = {
+    "CBCC_DATA_SOURCE_SEL": "ENCODED",
+    "RXBUF_ADDR_MODE": "FULL",
+    "RXOOB_CLK_CFG": "PMA",
+    "RXSLIDE_MODE": "OFF",
+    "RX_XCLK_SEL": "RXREC",
+    "SATA_PLL_CFG": "VCO_3000MHZ",
+    "TXPI_PPMCLK_SEL": "TXUSRCLK",
+    "TX_DRIVE_MODE": "DIRECT",
+    "TX_XCLK_SEL": "TXOUT",
+}
+
+
+def get_gtp_channel_site(db, grid, tile, site):
```

Comment:
This assumes there is only one GTP CHANNEL per tile, can you change the code to validate that assumption?  E.g. run the site search and make sure only one site is found.

Reply:
Done